### DISCUSSION TITLE
Welcome to skater-xl-mapping-wiki Discussions!

### DISCUSSION THREAD

**Discussion by SqueegeeDino** [SkaterXLModding/skater-xl-mapping-wiki#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SXL-SoXoS** [comment]
Hey dude. Is an update planned for the **map patch**? The whole community is eagerly waiting!
